### DISCUSSION TITLE
查询报错

### DISCUSSION THREAD

**Discussion by zqb666** [zuihou/zuihou-admin-cloud#224] [02.提问]
Caused by: net.sf.jsqlparser.parser.ParseException: Encountered unexpected token: "," ","
    at line 2, column 59.

Was expecting one of:

    ";"
    "CONNECT"
    "EMIT"
    "GROUP"
    "HAVING"
    "INTO"
    "START"
    "WINDOW"
    <EOF>
把该sql 拿到navicat正常可以运行 ，通过接口调用报上面的错
怀疑 是多租户插件引起 不知道什么原因 怎么解决
加了  @InterceptorIgnore(tenantLine = "true") 过滤多租户逻辑后不报错

**Comment by zuihou** [comment]
如果确实是插件的问题， 请到 mybatis plus 官方提issue （给人提问题时，请附上的sql，不然他会直接关你issue的）。

**Reply by zqb666** [reply]
感谢你的建议

**Reply by zqb666** [reply]
jsqlparser 我发现是缺少这个包引起的，奇怪了 ，我发现作者你的lam-util里有用到这个，但是没找到依赖的地方 你把这个maven依赖放在哪里了？

**Reply by zuihou** [reply]
用了COLUMN模式，就会默认引入 jsqlparser
